### ISSUE TITLE
Stadia maps typings have retina and apiKey as required parameters in typescript

### ISSUE THREAD

**Issue by Siedlerchr** [openlayers/openlayers#15232]
**Describe the bug**
Migrating my stamen to Stadia Maps: 
Using v8.1.0 in typescript I get the error message that:
```
Argument of type '{ layer: string; }' is not assignable to parameter of type 'Options'.
  Type '{ layer: string; }' is missing the following properties from type 'Options': apiKey, retina 
```

**To Reproduce**
Use the example in typescript: https://github.com/openlayers/openlayers/blob/v8.1.0/changelog/upgrade-notes.md 

**Expected behavior**
apiKey and retina should be optional parametersr

**Comment by ahocevar** [comment]
Thanks for reporting this, @Siedlerchr. This will be fixed with #15278

**Comment by Siedlerchr** [comment]
Thanks a lot. I will try it soon
